src/recipe/parser/build.rs

```diff
@@ -55,9 +101,16 @@ pub struct Build {
     /// Setting to control wether to always include a file (even if it is already present in the host env)
     #[serde(default, skip_serializing_if = "GlobVec::is_empty")]
     pub(super) always_include_files: GlobVec,
+    /// Variant ignore and use keys
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
I prefer to have an `is_default` on the `VariantKeyUsage` and get rid of the `Option`.

Reply:
We can use `std::ops::Not::not`

Reply:
You didn't push any changes if I see correctly.